app/sim_test.go

```diff
@@ -88,7 +87,7 @@ func TestAppStateDeterminism(t *testing.T) {
 	config.ExportParamsPath = ""
 	config.OnOperation = false
 	config.AllInvariants = false
-	config.ChainID = helpers.SimAppChainID
+	config.ChainID = "gaia-app"
```

Comment:
is this intended to be merged in?

Reply:
Yes, the updates to SDK mean that old Gaia deps break everything. Hard coding that simapp value lets us get rid of github.com/cosmos/gaia/v4/app/helpers

I'm all ears for better solutions, but given we don't really use simulation at all I just went for quick.